src/utils/formatStatusToTitleCase.ts

```diff
@@ -0,0 +1,8 @@
+export function formatStatusToTitleCase(status: string) {
+  const words = status.split("_");
+  const formattedStatus = words
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase())
+    .join(" ");
+
+  return formattedStatus;
+}
```

Comment:
not very intuitive what this particular function does. Can we make it more readable?